javascript-light-bulb/main.js

```diff
@@ -0,0 +1,26 @@
+// button class="bulb-on" //
+// main class="background-on" //
+
+// use doc.querySelector to get bulb-on in html and assign to var button //
+// have var userclicks store # of clicks //
+// create a function lightSwitch that changes on/off //
+// if userclicks is divisible by 2, bulb is on and background is on//
+// if userclicks is not divisible by 2, bulb is off and background is off//
+// every time button is clicked, start the function //
+
+var button = document.querySelector('.bulb-on');
+var ground = document.querySelector('.background-on');
+var userClicks = 0;
+
+function lightSwitch(event) {
+  userClicks++;
+  if (userClicks % 2 === 0) {
```

Comment:
Instead of using a counter variable and just checking if the current count is even or odd, there is a better data type that has only two states that you can use to determine if the light is "on" or not: a boolean.